opennms-base-assembly/src/main/filtered/etc/service-configuration.xml

```diff
@@ -172,15 +172,7 @@ maintained
   </service>
   <service>
     <name>OpenNMS:Name=Provisiond</name>
-    <class-name>org.opennms.netmgt.daemon.SimpleSpringContextJmxServiceDaemon</class-name>
-    <attribute>
-      <name>LoggingPrefix</name>
-      <value type="java.lang.String">provisiond</value>
-    </attribute>
-    <attribute>
-      <name>SpringContext</name>
-      <value type="java.lang.String">provisiondContext</value>
-    </attribute>
+    <class-name>org.opennms.netmgt.provision.service.jmx.Provisiond</class-name>
```

Comment:
We want to avoid changing configuration files in `foundation-2016`, so if this change is required, we may want to consider re-targeting this towards `foundation-2017` instead.